moped-database/migrations/1650301717000_alter_project_list_view/up.sql

```diff
@@ -0,0 +1,89 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
+with moped_team_members as (
+  select
+    mpp.project_id,
+    string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ',') AS project_team_members
+   FROM 
+    public.moped_proj_personnel mpp
+        LEFT JOIN public.moped_users mu ON ((mpp.user_id = mu.user_id))
+        LEFT JOIN public.moped_project_roles mpr ON ((mpp.role_id = mpr.project_role_id))
```

Comment:
Can we be assured that the records in `moped_users` and `moped_project_roles` are going to be available? If so, you may consider using a vanilla `JOIN`.

Reply:
I think we can be assured the records will be available. ill update to using join, im going to read up on why using `join` is preferable to `left join` in this instance.

Reply:
I think a `left join` and a `join` here are going to do the exact same operation. It's kind of like extra parentheses -- if you don't need them, it can be argued that it's clearer to not include them. One thing for sure though, this is a very minor issue and since it's working, it's a good solution!